sys/net/application_layer/nanocoap/sock.c

```diff
@@ -37,6 +39,13 @@
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
+/**
+ * @brief   Size of the buffer used for the DTLS handshake
+ */
+#ifndef CONFIG_NANOCOAP_DTLS_HANDSHAKE_BUF_SIZE
+#define CONFIG_NANOCOAP_DTLS_HANDSHAKE_BUF_SIZE (160)
```

Comment:
Did you find a hint for this size?

Reply:
No just trial & error / binary search

Reply:
some hint like 

```
/** 
 * this size was found suitable for DTLS using a simple PSK in mode AES_128_CCM_8 
 * DTLS places no restriction on its handshake package size therefor this might need change,
 * if mode or key-size change especially if certificates instead of PSK are used
 */
```

https://datatracker.ietf.org/doc/html/rfc4347#section-3.2.3

Reply:
added